raiden_contracts/tests/test_print_gas.py

```diff
@@ -316,11 +318,22 @@ def print_gas_one_to_n(
     get_private_key: Callable,
     web3: Web3,
     print_gas: Callable,
+    service_registry: Contract,
+    custom_token: Contract,
 ) -> None:
     """ Abusing pytest to print gas cost of OneToN functions """
     (A, B) = get_accounts(2)
     deposit_to_udc(A, 30)
 
+    # B registers itself as a service provider
+    deposit = service_registry.functions.currentPrice().call()
```

Comment:
This still seems duplicated:
- https://github.com/raiden-network/raiden-contracts/pull/1151/files#diff-a0ec52e4288fe8c7b94e7d1fc9704dc6R397
- https://github.com/raiden-network/raiden-contracts/pull/1151/files#diff-7ecca76ac1d6a1de743191515e65a0e0R62

Not sure what your general approach is in this repo, as it is only 3 lines...

Reply:
The general approach is to reuse the fixture.  A fix is coming in this PR.

Reply:
I've removed this part.

There is something similar in the same file, but there the duplication is harder to remove.  I need the gas consumption of the deposit transaction.